megamek/src/megamek/common/Entity.java

```diff
@@ -12983,12 +12983,27 @@ public int getRearArc() {
         return Compute.ARC_REAR;
     }
 
+    public static class SensorRangeHelper {
```

Comment:
Entity is already quite long. Feel free to place such classes into their own file and make them immutable if possible.